ext/js/pages/settings/dictionary-controller.js

```diff
@@ -822,6 +824,36 @@ export class DictionaryController {
         }
     }
 
+    /**
+     * @param {import('settings').ProfileOptions} options
+     */
+    _updateCurrentEntries(options) {
+        const dictionariesModalBodyScrollY = this._dictionaryModalBody.scrollTop;
+        const dictionaries = this._dictionaries;
+        if (dictionaries === null) { return; }
+
+        for (const entry of this._dictionaryEntries) {
+            entry.cleanup();
+        }
+
+        /** @type {Map<string, import('dictionary-importer').Summary>} */
+        const dictionaryInfoMap = new Map();
+        for (const dictionary of dictionaries) {
+            dictionaryInfoMap.set(dictionary.title, dictionary);
+        }
+
+        const dictionaryOptionsArray = options.dictionaries;
```

Comment:
Add a comment here saying what this for loop is doing?

Reply:
Made the loop definition more sane. Should be clear what it does now.